deployment/terraform/modules/osv/scripts/gcloud_build_image

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
```

Comment:
CAn you link to https://github.com/GoogleCloudPlatform/esp-v2/blob/master/docker/serverless/gcloud_build_image here as well?